### PR TITLE
docs/: add files with test plan for platform provisioning and attestation module

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/attestation_tests.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Attestation
+
+Test plan - module attestation
+
+## /attest
+
+1. Singing metadata
```
Comment (sulewskiprzemyslaw) [inline docs/attestation_tests.md]: You may have meant:  "Signing?"

File: docs/attestation_tests.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+1. Input handling - metadata signed with `fobnail_ak.key`
+1. Send a query - add an endpoint in the dictionary
+1. Return payload contains PCR selection and nonce
+1. Output handling with TPM2_Quote() - add function in python
+
+## /attest/{id}
+
+1. id = `PC_ID` - result of `/admin/provision`
+1. Input handling - result of TPM2_Quote()
+1. Send a query - add an endpoint in the dictionary
```
Comment (sulewskiprzemyslaw) [inline docs/attestation_tests.md]: Please also add dots at the end of the phrases.

File: docs/platform_provisioning_tests.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+1. id = `PC_ID`
+1. Singing metadata
+1. Input handling - hardcoded metadata signed with `fobnail_ak.key`
+1. Send a query - add an endpoint in the dictionary
+
+## /admin/provision/{id}
+
+1. id = `PC_ID`
+1. Send a query (no payload) - add an endpoint in the dictionary
```
Comment (sulewskiprzemyslaw) [inline docs/platform_provisioning_tests.md]: Please also add dots at the end of the phrases.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/attestation_tests.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/attestation_tests.md]: Yeah, my bad, corrected

File: docs/attestation_tests.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/attestation_tests.md]: added

File: docs/platform_provisioning_tests.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/platform_provisioning_tests.md]: added

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**